contracts/DelayModule.sol

```diff
@@ -63,89 +36,45 @@ contract DelayModule {
     /// @param expiration Duration that a proposed transaction is valid for after the cooldown, in seconds (or 0 if valid forever)
     /// @notice There need to be at least 60 seconds between end of cooldown and expiration
     function setUp(
-        Executor _executor,
+        address _executor,
         uint256 cooldown,
         uint256 expiration
     ) public {
-        require(
-            address(executor) == address(0),
-            "Module is already initialized"
-        );
+        require(executor == address(0), "Module is already initialized");
         require(
             expiration == 0 || expiration >= 60,
             "Expiratition must be 0 or at least 60 seconds"
         );
-        if (address(_executor) != address(0)) setupModules();
 
         executor = _executor;
         txExpiration = expiration;
         txCooldown = cooldown;
 
-        emit DelayModuleSetup(msg.sender, address(_executor));
-    }
-
-    modifier executorOnly() {
-        require(msg.sender == address(executor), "Not authorized");
-        _;
-    }
-
-    modifier moduleOnly() {
-        require(modules[msg.sender] != address(0), "Module not authorized");
-        _;
+        if (_executor != address(0)) setupModules();
+        emit DelayModuleSetup(msg.sender, _executor);
     }
 
     function setupModules() internal {
         require(
             modules[SENTINEL_MODULES] == address(0),
             "setUpModules has already been called"
         );
+        transferOwnership(executor);
```

Comment:
We should pass in the owner variable to `setup()`, rather than transferring to executor. Since it's possible that these two might be different addresses.

For example, if this modifier were enabled on another modifier, rather than an executor.
Or of the DAO wants some other entity to hold its owner permissions.